scripts/helpers.js

```diff
@@ -0,0 +1,64 @@
+import { months, directions } from "./data";
+
+export function createDOMElement(tagName, ...classNames) {
+    const element = document.createElement(tagName);
+    element.classList.add(...classNames);
+    return element;
```

Comment:
blank line before return
```suggestion

    return element;
```